backend/mlarchive/bin/compare.py

```diff
@@ -21,53 +21,58 @@
 from mlarchive.bin.scan_utils import all_mboxs
 from mlarchive.archive.models import Message
 from mlarchive.archive.mail import archive_message, MessageWrapper
+from zoneinfo import ZoneInfo
 
 TOTAL = 0 
 MISSING = 0
 IMPORTED = 0
 
+pacific = ZoneInfo('US/Pacific')
```

Comment:
Be ready to change all of the notion of when we do things to be UTC

Reply:
Using Pacific timezone here to match the timezone of the server OS / filesystem. Are you saying this will change to UTC? With new infrastructure?